confgenerator/confgenerator_test.go

```diff
@@ -172,18 +172,18 @@ func generateConfigs(platform platformConfig, testDir string) (got map[string]st
 			got["error"] = err.Error()
 		}
 	}()
-	// Merge Config
-	builtInConfBytes, confBytes, err := confgenerator.MergeConfFiles(
+
+	uc, err := confgenerator.MergeConfFiles(
 		filepath.Join("testdata", testDir, inputFileName),
 		platform.OS,
 		apps.BuiltInConfStructs,
 	)
 	if err != nil {
 		return
 	}
-	got[builtinConfigFileName] = string(builtInConfBytes)
+	got[builtinConfigFileName] = apps.BuiltInConfStructs[platform.OS].String()
```

Comment:
Maybe built in config could be a field on the Unified Config instead of always needing to look up from `apps`? That or `apps` could have a public function that gets the built in configuration for that platform.